test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -518,6 +519,24 @@ pub fn test_create_account_failure_invalid_name(node: impl Node) {
     }
 }
 
+pub fn test_create_account_failure_no_funds(node: impl Node) {
+    let account_id = &node.account_id().unwrap();
+    let node_user = node.user();
+    let transaction_result = node_user
+        .create_account(account_id.clone(), eve_dot_alice_account(), node.signer().public_key(), 0)
+        .unwrap();
+    assert_eq!(
+        transaction_result.status,
+        FinalExecutionStatus::Failure(TxExecutionError::ActionError(ActionError {
+            index: None,
+            kind: LackBalanceForState {
+                account_id: eve_dot_alice_account(),
+                amount: 16543800000000000000000
```

Comment:
We shouldn't hardcode the amount unless you use it within fixed fee test, otherwise it would break after every fee change and someone else would have to fix the test.
You can use the following: https://github.com/nearprotocol/nearcore/blob/master/test-utils/testlib/src/standard_test_cases.rs#L1047-L1057